code/script.js

```diff
@@ -15,14 +189,17 @@ const showMessage = (message, sender) => {
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/anime.png" alt="User" />  
       </section>
     `
+    //Above, we get ahold of chat and add som innerHTML using += (add something to it). The show message function will be re-used several times. The innerHTML section will be added to the chat-section in the HTML-dokument. 
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
+  } 
+  else if (sender === 'bot') {
+    
     chat.innerHTML += `
```

Comment:
If you want to save time and set a delay on every bot/user message you could add the setTimeout to the else/if statements like this this:

```javascript
else if (sender === 'bot') {
setTimeout(() => {
    chat.innerHTML += `
      <section class="bot-msg">
        <img src="assets/bot.png" alt="Bot" />
        <img src="assets/kawaii.png" alt="Kawaii" />
        <div class="bubble bot-bubble">
          <p>${message}</p>
        </div>

  chat.scrollTop = chat.scrollHeight;
}, 1000);
}
```

Reply:
Oh yeah, good idea! Thanks for the tip!